pages/recording/[[...id]].tsx

```diff
@@ -169,7 +169,7 @@ function RecordingPage({
     return (
       <>
         {head}
-        <LoadingScreen />
+        <LoadingScreen fallbackMessage="Finding recording..." />
```

Comment:
I actually don't love showing these messages quickly and then replacing them because it is low signal to noise, but happy to land this and then do a second pass on the loader after we know more about the expected perf.